prow/config/config.go

```diff
@@ -1132,7 +1171,10 @@ func yamlToConfig(path string, nc interface{}) error {
 		jc = v
 	case *Config:
 		jc = &v.JobConfig
+	default:
```

Comment:
Could you add a comment explaining why return nil instead of error?

Reply:
Done